docs/src/pages/ui/theme/alternativeStyling/index.page.mdx

```diff
@@ -0,0 +1,501 @@
+
+const useStyles = createUseStyles({
+  styledText: {
+    color: 'blue',
+  },
+});
+
+const classes = useStyles();
+
+<Text className={classes.styledText} color="red">
+  This is my JSS Text{' '}
+</Text>;
+```
+
+In the example above, the color of the view will be set to `red` because of the Amplify styling prop. [Here]() is a list of all Amplify styling props.
+
+##### Amplify variation props
+
+These are Amplify props that are added to certain Amplify components to change the look and/or behavior of the Amplify component. Examples of these variation props are `size` or `variation`.
+
+```jsx
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import { Button } from '@aws-amplify/ui-react';
+
+const useStyles = createUseStyles({
+  styledButton: {
+    color: 'blue',
+  },
+});
+
+const classes = useStyles();
+
+<Button className={classes.styledButton} variation="primary">
+  Primary Button
+</Button>;
+```
+
+In the example above, the color of the button will be set to `white`, because the primary variation of the button has a color set to `white` and it takes precedence over the JSS styling. A list of Amplify variations can be found on [here]().
+
+##### Amplify Custom ClassNames
+
+These are custom classnames added onto Amplify components and can be used with CSS styling rules to modify the look of an Amplify component.
+
+```jsx
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import { View } from '@aws-amplify/ui-react';
+
+const useStyles = createUseStyles({
+  styledText: {
+    color: 'blue',
+  },
+});
+
+const classes = useStyles();
+const classNames = `${classes.styledText} my-styled-text`;
+
+<Text className={classNames}>This is my JSS Text </Text>;
+```
+
+```CSS
+/* External Style Sheet */
+.my-styled-text {
+  color: red;
+}
+```
+
+In the example above, the color of the text will be `blue` because the JSS styling will take precedence over the classname CSS.
+
+##### CSS Root Variable Overrides
+
+Amplify theming provides many CSS variables to customize the look and feel of the entire application. These CSS variables can be used at a root level to override all instances of component styling.
+
+```jsx
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import { Button } from '@aws-amplify/ui-react';
+
+const useStyles = createUseStyles({
+  styledButton: {
+    color: 'blue',
+  },
+});
+
+const classes = useStyles();
+
+<Button className={classes.styledButton}>Button</Button>;
+```
+
+```CSS
+/* External Style Sheet */
+:root {
+  --amplify-components-button-color: red;
+}
+```
+
+In the example above, the button color will be set to `blue` because the JSS styling will take precedence over the Amplify CSS variable.
+
+</Block>
+
+<Block name="Emotion">
+[Emotion](https://github.com/emotion-js/emotion) is a performant and flexible CSS-in-JS library. It allows you to style apps quickly with string or object styles.
```

Comment:
We might want to be a bit more objective in these descriptions so it doesn't look like we are endorsing one library over another.

Reply:
Yeah probably, for the most part I just took the first sentance or two of their description in their readme file and that is what I used for the description. I'll rework this one to be more objective.